src/dotnet/Chat.UI.Blazor/Components/ChatNotificationToggle.razor

```diff
@@ -0,0 +1,48 @@
+@namespace ActualChat.Chat.UI.Blazor.Components
+@using ActualChat.Notification
+@inherits ComputedStateComponent<ChatNotificationToggle.Model>
+
+@{
+    var isSubscribed = State.Value.IsSubscribed;
+    var toggleClass = isSubscribed
+        ? "text-toggle-on"
```

Comment:
We've agreed that we'll be adding `on` class for items in "On" state, so the selector will be ~ `.toggle.on`

Reply:
I don't want to create new styles now, so I have to use existing.

Reply:
I mean when Andrey will refactor existing styles to that approach I can do the same